test/core/security/roleRepository.test.js

```diff
@@ -488,6 +488,8 @@ describe('Test: security/roleRepository', () => {
     const { NativeController } = require('../../../lib/api/controller/base');
 
     beforeEach(() => {
+      kuzzle.ask = sinon.stub();
```

Comment:
Always stub with `sinon.stub(<object>, "<method name>")`: that way, if the method doesn't exist anymore, or if it is already stubbed, sinon will complain and tests will fail, which might prevent false positive and is more secure.
Also, this allows restoring the stubbed method with `<stubbed method>.restore()`

And in this case, `kuzzle.ask` is already stubbed in KuzzleMock :wink: